### PR TITLE
VS-13500 handling responses with no safe search data

### PR DESCRIPTION
**Pull request by artur-dev**
Throwing VisionClientException when:
- Google API returned response with error message,
- Google API returned response without safe search data or error message.

### CODE REVIEW

File: src/VisionClientException.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?php
+namespace Vivastreet\GoogleCloud\Vision;
+
+class VisionClientException extends \Exception
```
Comment (ybenzine) [inline src/VisionClientException.php]: This is fine as-is. However, can we expect more exceptions in that `GoogleCloud\Vision ` namespace?

Basically my question is, would there be a benefit to having a `GoogleCloudException` or `VisionException` to extend from in the future? Or do you think that's overkill?

Comment (artur-dev) [inline src/VisionClientException.php]: At the moment I don't think we are going to provide more exceptions.

Comment (jpcfernandes) [inline src/VisionClientException.php]: I think we ain't gonna need it.

Comment (mrbadguy78) [inline src/VisionClientException.php]: 👍

**Pull request merged**